### ISSUE TITLE
No module named 'find_delay' on 2.13

### ISSUE THREAD

**Issue by denkorz** [RomainPastureau/find_delay#7]
Error when the latest version (2.13) is used: _...\analyseAudio.py", line 1, in <module>
    from find_delay import find_delay
ModuleNotFoundError: No module named 'find_delay'_

This file starts with 
```
from find_delay import find_delay
import sys
import logging

audio1 = sys.argv[1]
audio2 = sys.argv[2]
plot = sys.argv[3]
```

When I run `pip3 install find-delay==2.12`, everything works correctly.
To see the error, it's enough just to run  `python -u "c:\...\analyseAudio.py"`

I switched from Mac to Windows and needed to configure the env. To fix this issue, I tried reinstall Python, rewrite Path, use full pathes to executables and so on. I'm not sure the issue is related to your update.
But after I tried everything and spent all the free chatGPT limits I tried to check the lib version. It was new. And after I downgraded the lib, everything started to work.

If it's related to me or my env, just close this issue :)

**Comment by RomainPastureau** [comment]
Hi! Yes, I just tested it myself... The module does not import, which is very weird.
I'll investigate and let you know in this thread. Thanks for letting me know!

**Comment by RomainPastureau** [comment]
Hi again,

I found the problem (I think) - which stemmed from the creation of the wheel and tar.gz archives. For 2.13 I had switched from hatchling to setuptools, because trying to upload the package with twine would return an error message. It turns out, I needed to update the pkginfo module. If you didn't understand anything reading this, that makes two of us.

No matter what, I just uploaded a version 2.14, and yanked the 2.13. Let me know if this new version works for you, and sorry for the issue...
